animations/lib/src/misc/physics_card_drag.dart

```diff
@@ -44,7 +44,7 @@ class _DraggableCardState extends State<DraggableCard>
   /// this value is set to the value of the [_animation].
   var _dragAlignment = Alignment.center;
 
-  Animation<Alignment> _animation;
+  Animation<Alignment>? _animation;
```

Comment:
could this be `late`?

Reply:
Now that I'm looking at this again, I think you're right.